coldfront/core/allocation/templates/allocation/allocation_table_view.html

```diff
@@ -0,0 +1,158 @@
+            <a href="?order_by=service_rate&direction=des&{{filter_parameters}}"><i class="fas fa-sort-down" aria-hidden="true"></i><span class="sr-only">Sort Service Rate desc</span></a>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for allocation in allocation_list %}
+          <tr>
+            <td><a href="/qumulo/allocation/{{allocation.id}}">{{ allocation.id }}</a></td>
+            <td class="text-nowrap"><a 
+                href="/project/{{allocation.project_id}}/">{{ allocation.project_name|truncatechars:50 }}</a></td>
+            <td class="text-nowrap">{{allocation.pi_first_name}} {{allocation.pi_last_name}}
+              ({{allocation.pi_user_name}})</td>      
+              <!-- Replace with new columns -->
+            <td class="text-nowrap">{{ allocation.resource_name }}</td>
+            <td class="text-nowrap">{{ allocation.allocation_status }}</td>
+            <td class="text-nowrap">{{ allocation.department_number }}</td>
+            <td class="text-nowrap">{{ allocation.itsd_ticket }}</td>
+            <td class="text-nowrap">{{ allocation.file_path }}</td>
+            <td class="text-nowrap">{{ allocation.service_rate }}</td>
+          </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+    {% if is_paginated %} Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}
+      <ul class="pagination float-right mr-3">
+        {% if page_obj.has_previous %}
+          <li class="page-item"><a class="page-link" href="?page={{ page_obj.previous_page_number }}&{{filter_parameters_with_order_by}}">Previous</a></li>
+        {% else %}
+          <li class="page-item disabled"><a class="page-link" href="#">Previous</a></li>
+        {% endif %}
+        {% if page_obj.has_next %}
+          <li class="page-item"><a class="page-link" href="?page={{ page_obj.next_page_number }}&{{filter_parameters_with_order_by}}">Next</a></li>
+        {% else %}
+          <li class="page-item disabled"><a class="page-link" href="#">Next</a></li>
+        {% endif %}
+      </ul>
+    {% endif %}
+  </div>
+{% elif expand_accordion == "show"%}
+  <div class="alert alert-secondary">
+    No search results!
+  </div>
+{% else %}
+  <div class="alert alert-secondary">
+    No allocations to display!
+  </div>
+{% endif %}
+
+<script>
+  $("#navbar-main > ul > li.active").removeClass("active");
+  $("#navbar-project-menu").addClass("active");
+  $("#navbar-allocation").addClass("active");
+
+  $(document).on('click', '#form_reset_button', function () {
+    resetForm($('#filter_form'));
+  });
+
+  $(".datepicker").flatpickr();
+
+  function resetForm($form) {
+    $form.find('input:text, input:password, input:file, select, textarea').val('');
+    $form.find('input:radio, input:checkbox').removeAttr('checked').removeAttr('selected');
+  };
+
+  $("#expand_button").click(function () {
```

Comment:
for consistency with existing code, I'm requesting that we use arrow functions where possible:

```suggestion
  $("#expand_button").click(() => {
```